### PR TITLE
MINOR: [R][Docs] Add note about RSPM binary installation

### PR DESCRIPTION
**Pull request by eitsupi**

### CODE REVIEW

**Comment by pitrou** [comment]
@assignUser Can you review this PR?

**Review by assignUser — CHANGES_REQUESTED** [review]
I tested the user agent vs. the script rstudio provides ✔️  
Just one small comment :)

File: r/vignettes/install.Rmd [hunk 1]
```diff
@@ -43,10 +43,14 @@ If you want a quicker installation process, and by default a more fully-featured
 
 For example, if you are using Ubuntu 20.04 (Focal):
 
-```{r, eval = FALSE}
+```r
+options(HTTPUserAgent = sprintf("R/%s R (%s)", getRversion(), paste(getRversion(), R.version["platform"], R.version["arch"], R.version["os"])))
```
[the remark this answers is not in the document]
Comment (eitsupi) [reply r/vignettes/install.Rmd:47]: Perhaps a tab size of 2 is more appropriate than 4?

[the remark this answers is not in the document]
Comment (assignUser) [reply r/vignettes/install.Rmd]: (I edited the suggestion ;) )

**Review by assignUser — APPROVED** [review]
👍  looks good to me @pitrou

**Pull request closed without merging**